wdio-chrome.conf.js

```diff
@@ -189,8 +186,29 @@ exports.config = {
      * @param {Array.<Object>} capabilities list of capabilities details
      * @param {Array.<String>} specs List of spec file paths that are to be run
      */
-    // before: function (capabilities, specs) {
-    // },
+    before: function (capabilities, specs) {
+        browser.addCommand('percyScreenshot', (screenshotName, featureType) => {
+
+            let viewportWidths = [];
```

Comment:
Same as before. No need to initialise to `[]`, unless you're planning to add multiple elements, in which case then you could do `viewportWidths.push(...)`.